components/dash-core-components/src/components/Loading.react.js

```diff
@@ -22,8 +22,8 @@ const getSpinner = spinnerType =>
 const Loading = ({
     children,
     loading_state,
-    display,
-    color,
+    display = 'auto',
+    color = '#119DFF',
```

Comment:
do we have colors and other constants scattered elsewhere through the code as well or is there a way to consolidate them all in one place? (very low priority, not worth holding up this PR for)